storage/resource_mount_test.go

```diff
@@ -108,6 +108,78 @@ func TestResourceAwsS3MountGenericCreate(t *testing.T) {
 	assert.Equal(t, testS3BucketPath, d.Get("source"))
 }
 
+func TestResourceAwsS3MountGenericCreateWithInvalidClusterId(t *testing.T) {
```

Comment:
This case is not affected by your PR, right?

Reply:
Yup